spec/fixtures/authorities/linked_data/lod_encoding_config.json

```diff
@@ -1,4 +1,5 @@
 {
+  "QA_CONFIG_VERSION": "2.0",
```

Comment:
This and all other files in `spec/fixtures/authorities/linked_data` are being updated to be linked data config version 2.0 compliant.